src/components/FloatingActionButton.tsx

```diff
@@ -69,9 +70,11 @@ function FloatingActionButton({onPress, isActive, accessibilityLabel, role}: Flo
     const isNarrowScreenOnWeb = shouldUseNarrowLayout && platform === CONST.PLATFORM.WEB;
     const isFocused = useBottomTabIsFocused();
     const [isSidebarLoaded] = useOnyx(ONYXKEYS.IS_SIDEBAR_LOADED, {initialValue: false});
+    const isActiveRouteHome = useIsCurrentRouteHome();
```

Comment:
NAB, but could we use the same terminology here? Either active route or current route. 
```suggestion
    const isActiveRouteHome = useIsCurrentRouteHome();
```